widget/ui/src/components/Flags/Flag.tsx

```diff
@@ -0,0 +1,25 @@
+import type { FlagPropTypes } from './Flags.types';
+import type { PropsWithChildren } from 'react';
+
+import React from 'react';
+
+export function Flag(props: PropsWithChildren<FlagPropTypes>) {
```

Comment:
I'm not sure making a separate component for `svg` here is necessary. I suggest to use `<svg ...>` right inside the component itself. 
I suggest this structure for this component:
```

/Flags
    index.ts -> only `export { default as English }`. you already did it.
    Flags.constants.ts -> `export const DEFAULT_SIZE = "1em"`
    Flags.types.ts -> keep your current file, using same interface for flags is a good idea .
    English.tsx -> equivalent to your `EnFlag.tsx` 
    ... -> Other languages

```

Note: Please use full language name for your files.